src/Quartz.Tests.Unit/Collections/OrderedDictionaryTest.cs

```diff
@@ -0,0 +1,36 @@
+using NUnit.Framework;
+using Quartz.Collections;
+using System;
+
+namespace Quartz.Tests.Unit.Collections
+{
+    [TestFixture]
```

Comment:
This has been  redundant for a good while now, Test attribute is enough for discovery.

Reply:
I'm getting out of touch, it seems. Thanks for the info.